src/test/java/de/tum/in/www1/artemis/programmingexercise/ProgrammingExerciseIntegrationBambooBitbucketJiraTest.java

```diff
@@ -71,6 +71,12 @@ void testProgrammingExerciseIsReleased_forbidden() throws Exception {
         programmingExerciseIntegrationTestService.testProgrammingExerciseIsReleased_forbidden();
     }
 
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
+    void testExportSubmissionsByParticipationIds_excludePracticeSubmissions() throws Exception {
+        programmingExerciseIntegrationTestService.testExportSubmissionsByParticipationIds_excludePracticeSubmissions();
+    }
+
```

Comment:
I think you also need to add this to `ProgrammingExerciseIntegrationJenkinsGitlabTest`

Reply:
You are right. I saw it before and thought about it but somehow came to the conclusion that it' not necessary... Thanks for pointing that out :)